tooling/specs/specs.go

```diff
@@ -56,11 +82,23 @@ func (s Spec) IsEnabled() bool {
 }
 
 func (s Spec) Enable() {
-	specEnabled[s] = true
+	if specs, ok := specGroups[s]; ok {
+		for _, s := range specs {
+			specEnabled[s] = true
```

Comment:
You might need a recursive call here since we don't differentiate between leaves and groups: typing allows for dags of specs.

It might be nice to have a type for leaves and one for leaves+groups so we can enforce my point below regarding internal/external APIs.